src/lib/moment/momentUtils.ts

```diff
@@ -6,6 +6,18 @@ export const formatUTCDateStringToLocal = (dateStringUTC: string): string => {
   return moment(dateStringUTC).local().format('YYYY-MM-DD HH:mm:ss');
 };
 
+
+export const lessThanCurrentPeriod = (dateString: string): boolean => {
+  const currentDate = moment(dateString);
+  const currentMonth = moment().startOf('month');
+  const currentDatePeriod = currentDate.valueOf();
+  const currentPeriod = currentMonth.valueOf();
```

Comment:
Naming is confusing here. It makes it especially hard to follow because `currentMonth` refers to the actual current month, but `currentDate` refers to the date given after parsing the date string (which could be past dates). 

We need a better name than `currentDate` and etc so that the logic is easy to read and understand